cdec/installation-manager-core/src/main/java/com/codenvy/im/managers/helper/NodeManagerHelperCodenvy3Impl.java

```diff
@@ -229,8 +228,8 @@ public Command getUpdatePuppetConfigCommand(String oldHostName, String newHostNa
     }
 
     @Override
-    public void validateLicense() {
-        // Codenvy 3.x doesn't support licenses
+    public boolean isValidLicense() {
+        throw new UnsupportedOperationException("Codenvy 3.x doesn't support licenses");
```

Comment:
Please, restore initial version, because exception in this place breaks adding node in Codenvy 3.x